content/en/faq/setup/install-method-selection.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Which Istio installation method should I use?
+weight: 10
+---
+
+In addition to the simple [getting started](/docs/setup/getting-started) evaluation install, there are several different
+methods you can use to install Istio. Which one you should use depends on your production requirements.
+The following lists some of the pros and cons of each of the available methods:
+
+1. [istioctl install](/docs/setup/install/istioctl/)
+
+    The simplest and most qualified installation/management path with high security.
+    This is the community recommended method for most use cases.
+
+    Pros:
+
+    - Thorough configuration validation and health verification.
+    - Uses the `IstioOperator` API which provides extensive configuration/customization options.
+    - No in-cluster privileged pods needed. Changes are actuated by running the `istioctl` command.
+
+    Cons:
+
+    - Multiple binaries must be managed, one per Istio minor version.
+    - The `istioctl` command can set values like `JWT_POLICY` based on your running environment,
+      thereby producing varying installations in different Kubernetes environments.
```

Comment:
instead of `varying installations` - how about `producing environmentally optimized installations`.

This is shorter and cleaner.